schemas/com.snowplowanalytics.monitoring.batch/alert/jsonschema/1-0-0

```diff
@@ -36,10 +36,7 @@
     },
     "tags": {
       "description": "Set of key value pairs providing additional information",
-      "type": "object",
-      "additionalProperties": {
-        "type": "string"
-      }
+      "type": "object"
```

Comment:
@chuwy where is this schema used?  Assuming this cannot break any load processes / schema evolution through this removal?

Reply:
Saw the comment on the PR - no issues.